docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
 
   # Runs the database process:
   postgres:
-    image: postgres:10.6
+    image: postgres:12.3
```

Comment:
@wlonk Could you provide instructions for upgrading local data from 10.6 to 12.3? Or is it best just to `rm -rf ./postgres/` and start fresh?

Reply:
Usually I would use `pg_upgrade`, but combined with Docker, that's a bit thorny. Given that we have sample data scripts, etc, I think it might be just as useful to do the `rm` dance you describe; we don't treat our local dev data as valuable, after all.